packages/components/src/unit-control/stories/index.tsx

```diff
@@ -0,0 +1,162 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { UnitControl } from '../';
+import { CSS_UNITS } from '../utils';
+
+const meta: ComponentMeta< typeof UnitControl > = {
+	component: UnitControl,
+	title: 'Components (Experimental)/UnitControl',
+	argTypes: {
+		size: {
+			control: { type: 'select' },
+		},
+		onChange: {
+			control: { type: null },
+		},
+		onUnitChange: {
+			control: { type: null },
+		},
+	},
+	parameters: {
+		controls: {
+			expanded: true,
+			exclude: [ '__unstableInputWidth', '__unstableStateReducer' ],
+		},
+		docs: { source: { state: 'open' } },
+	},
+};
+export default meta;
+
+const DefaultTemplate: ComponentStory< typeof UnitControl > = ( {
+	onChange,
+	...args
+} ) => {
+	const [ value, setValue ] = useState< string | undefined >( '10px' );
+
+	return (
+		<div style={ { maxWidth: '100px' } }>
+			<UnitControl
+				{ ...args }
+				value={ value }
+				onChange={ ( v, extra ) => {
+					setValue( v );
+					onChange?.( v, extra );
+				} }
+			/>
+		</div>
+	);
+};
+
+export const Default: ComponentStory<
+	typeof UnitControl
+> = DefaultTemplate.bind( {} );
+Default.args = {};
```

Comment:
Should we include a label as part of the default args? I _think_ in most cases of our real world usage of the component, we typically include a label, so it might make a more consistent starting point.

Reply:
Fair point, I've added a label to the default args